### PR TITLE
🚮 Remove version-locking experiment

### PR DESCRIPTION
**Pull request by kristoferbaxter**
Remove `version-locking` experiment since it has been shipping for quite some time.

Closes https://github.com/ampproject/amphtml/issues/8236.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "description": "Fastish HTML",
   "main": "index.js",
   "engines": {
-    "node": "^10.0.0",
```
Comment (jridgewell) [inline package.json]: Undo please.

Comment (kristoferbaxter) [reply package.json]: Whoops! Thanks. Reverting this change.

File: build-system/global-configs/canary-config.json [hunk 2]
```diff
@@ -59,6 +59,5 @@
   "no-sync-xhr-in-ads": 1,
   "pump-early-frame": 1,
   "sandbox-ads": 1,
-  "user-error-reporting": 1,
-  "version-locking": 1
```
Comment (jridgewell) [inline build-system/global-configs/canary-config.json]: I think you have to make these in a separate PR.

Comment (kristoferbaxter) [reply build-system/global-configs/canary-config.json]: Will do.

**Comment by kristoferbaxter** [comment]
For some reason my commits are causing the Googlebot to think CLAs are invalid. I'll investigate after this issue is complete.

**Comment by mrjoro** [comment]
This PR was approved a while ago; should it be merged?

**Comment by kristoferbaxter** [comment]
Closing due to inactivity.
